src/common/createProgressNode.js

```diff
@@ -19,20 +20,22 @@ const createProgressNode = ({
   color,
   progress,
   progressBarBackgroundColor,
+  delay,
 }) => {
   const progressPercentage = clampValue(progress, 2, 100);
 
   return `
     <svg width="${width}" x="${x}" y="${y}">
       <rect rx="5" ry="5" x="0" y="0" width="${width}" height="8" fill="${progressBarBackgroundColor}"></rect>
-      <rect
-          height="8"
-          fill="${color}"
-          rx="5" ry="5" x="0" y="0"
-          data-testid="lang-progress"
-          width="${progressPercentage}%"
-      >
-      </rect>
+      <svg data-testid="lang-progress" width="${progressPercentage}%">
+        <rect
+            height="8"
+            fill="${color}"
+            rx="5" ry="5" x="0" y="0"
+            class="lang-progress"
+            style="animation-delay: ${delay}ms;"
+        />
+      </svg>
     </svg>
```

Comment:
Same here (see [this](https://github.com/anuraghazra/github-readme-stats/pull/2448/files#r1083277829)).